pkg/controller/node/nodecontroller_test.go

```diff
@@ -144,6 +144,11 @@ func TestMonitorNodeStatusEvictPods(t *testing.T) {
 		},
 	}
 
+	succPod := testutil.NewPod("pod0", "node0")
```

Comment:
Can you please create a separate test case for this? We're getting close to 1000-line test case, and I really don't want to get there...